stream_alert_cli/cli.py

```diff
@@ -232,7 +243,24 @@ def status(self):
             print '\n'
 
         print 'User access keys'
-        self._cmd(['terraform', 'output'])
+        self.run_command(['terraform', 'output'])
+
+    def rollback(self):
+        """Rollback the current production AWS Lambda version by 1
+
+        Notes:
+            Ignores if the production version is $LATEST
+            Only rollsback if published version is greater than 1
+        """
+        clusters = self.config['clusters'].keys()
+        for cluster in clusters:
+            current_vers = self.config['lambda_function_prod_versions'][cluster]
+            if current_vers != '$LATEST':
```

Comment:
can `current_vers` ever have the value of '1' or '0' and therefor potentially cause an issue here?